substrate-archive/src/database/models.rs

```diff
@@ -157,11 +178,13 @@ pub struct PersistentConfig {
 	pub minor: i32,
 	/// Patch version of this library.
 	pub patch: i32,
+	/// The chain data this db is populated with
+	pub chain: String,
```

Comment:
We should probably use genesis hashes for this rather than the chain name. Or a `(GenesisHash, String)` tuple to have the string for logs.

Reply:
added genesis_hash column